Godeps/Godeps.json

```diff
@@ -0,0 +1,6 @@
+{
+	"ImportPath": "k8s.io/test-infra",
```

Comment:
@spxtr will you review the vendor/godeps usage here? I think I'm leading him astray...